gratipay/billing/payday.py

```diff
@@ -27,7 +27,7 @@
 
 
 with open('sql/payday.sql') as f:
-    PAYDAY = f.read()
+    PAYDAY = f.read() % {'minimum_charge': MINIMUM_CHARGE}
```

Comment:
Ick. We moved away from extra interpolation in ... b045fb1f0edb4af19e289239ff691b8131d0f918 (#3785). Can we avoid reintroducing that here? I guess that would mean putting `minimum_charge` in the database somewhere.